### DISCUSSION TITLE
A bill payment was refunded but its amount still shows linked to the bill

### DISCUSSION THREAD

**Discussion by boris-bc** [firefly-iii/firefly-iii#4677] [Firefly III]
**Bug description**
I am running Firefly III version 5.5.1, and my problem is:

I made a transfer to pay a bill but this transfer was returned. I linked the refund transaction to the original payment. The refund transaction does not show linked to the bill so the bill still shows as paid. 

A workaround would be to be able to manually link the refund transaction to the bill but this seems not possible.

**Steps to reproduce**
- create an expense transaction that would be picked by a rule related to a bill
- create a revenue transaction and link it as a refund of the expense transaction from step one

**Reproduced in the Demo Instance**
Transactions:
https://demo.firefly-iii.org/transactions/show/457
https://demo.firefly-iii.org/transactions/show/458
Bill:
https://demo.firefly-iii.org/bills/show/1

**Comment by JC5** [comment]
Yes. The bill was still paid.

Transactions won't be able to cancel each other out. You could remove the original transaction.
